flask_resty_tenants/authorization.py

```diff
@@ -85,10 +85,10 @@ def check_request_tenant_id(self):
         try:
             tenant_id = self.get_request_tenant_id()
         except KeyError:
```

Comment:
Yes – it's distinguishing between `KeyError` and `None` – `KeyError` would mean that the request isn't supposed to have a `tenant_id`, v `None` meaning that it doesn't have one. It's like checking `undefined` v `null`.